parser/src/AST/Module.hs

```diff
@@ -60,6 +60,6 @@ data UserImport
 
 data ImportMethod = ImportMethod
     { alias :: Maybe (Comments, PreCommented String)
-    , exposedVars :: (Comments, PreCommented (Var.Listing Var.Value))
+    , exposedVars :: (Comments, PreCommented (Var.Listing Var.Value), Bool)
```

Comment:
Yes I was thinking if using newtype to make this more clear and avoid accidents might be a good idea but I didn't want to mix that into the work on this PR. If there is agreement on this I could create an issue for this and convert the projects multi line Bools to the new type